gunicorn/tests/compose/Dockerfile

```diff
@@ -6,4 +6,4 @@ RUN pip install gunicorn setproctitle
 COPY app.py /dummy_app.py
 
 EXPOSE 18000
-CMD gunicorn -w 2 --bind 0.0.0.0:18000 --name ${proc_name_env} dummy_app:app
+CMD gunicorn -w 4 --bind 0.0.0.0:18000 --name ${proc_name_env} dummy_app:app
```

Comment:
what is this change for?

Reply:
This increases the number of `workers` processes from 2 to 4.
The reason is just for e2e tests and "local integration tests" to assert the same metrics.